julia.version

```diff
@@ -0,0 +1,3 @@
+JULIA_BRANCH = add-makefile-support
+JULIA_SHA1 = d811d8a4a49b49c195aa28c5e46caa884a28aabf
```

Comment:
This is duplicate from the information in `Cargo.toml`. We should only keep one.

We can keep the info in either place. We can keep the information in `Cargo.toml`, and extract them and use in the makefile. Alternatively, we can keep the information in `julia.version`, and change `.github/scripts/ci-checkout.sh`.

Reply:
Fixed.